litmus-portal/frontend/src/pages/ChooseAndConfigureDashboards/index.tsx

```diff
@@ -1,8 +1,11 @@
 /* eslint-disable no-unused-expressions */
```

Comment:
Please remove the eslint disables, and try to fix those errors locally

Reply:
I have already restricted the usage of these to the places where its not possible to remove but will take a look.